### PR TITLE
Wiki pages containing question marks in their name weren't loading

### PR DESCRIPTION
**Pull request by Jofkos**
(untested)

### CODE REVIEW

File: models/wiki.go [hunk 1]
```diff
@@ -13,6 +13,8 @@ import (
 	"strings"
 	"sync"
```
Comment (Unknwon) [inline models/wiki.go]: Please remove this empty line.

**Comment by Unknwon** [comment]
Thanks your PR.

But build is failling.

**Comment by Jofkos** [comment]
Updated, but i still don't know if it is working.

**Comment by Unknwon** [comment]
OK... you have to make sure it did fix the problem you have.

**Comment by Jofkos** [comment]
I'm not able to build it, but if it's not the solution you'd like to have to fix that, it would be nice to somehow add support for page titles with question marks, or to forbid those

**Comment by Unknwon** [comment]
Oh... that's too bad... Let me see...

**Comment by Jofkos** [comment]
I was able to build it yesterday and i figured out that the solution works partially. With the changes you're able to open pages with names containing question marks, but you're not able to edit those.
I wasn't able to figure out why so i hope you will ^^

**Comment by Unknwon** [comment]
OK, thank you!

**Pull request merged**